src/index.js

```diff
@@ -3,12 +3,18 @@ import ReactDOM from 'react-dom';
 import {CommentCard} from './components/CommentCard';
 import {GBCard10} from './components/GBCard10';
 import {GBCard11} from './components/GBCard11';
+import {Button} from "./components/Button";
+import "./style/gb-style.css";
+import "./style/styleguide-presentation.css";
 
 class App extends React.Component {
     render() {
         return (
             <div>
                 <h1>Globuzzer Components</h1>
+                <h3>Buttons</h3>
+                <Button link="#" classes="gb-btn gb-btn-small gb-btn-primary">Button</Button>
```

Comment:
you can skip the gb-btn, because is used in all the buttons so you can add in in the Button contructor